bridge/bridge/analytics.py

```diff
@@ -0,0 +1,122 @@
+import os
+import hashlib
+from mixpanel import Mixpanel  # type: ignore
+from typing import Dict
+from bridge.deploy import DeployTarget
+import logging
+
+logger = logging.getLogger(__name__)
+
+# NOTE: client will throw an error if key is missing
+analytics_optout = os.environ.get("BRIDGE_ANALYTICS_OPT_OUT")
+mp_api_key = os.environ.get("MIXPANEL_API_KEY")
```

Comment:
Will we bake the actual value here into the Docker image? Could they opt out by unsetting that?

Reply:
I think setting an env like BRIDGE_ANALYTICS_OPT_OUT is better UX than unsetting the thing that will be baked into the image. Also doesn't call attention to the name of the env var they could use to extract the api key from the image